examples/basic/tidb-cluster.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: pingcap.com/v1alpha1
+kind: TidbCluster
+metadata:
+  name: basic
+spec:
+  version: v3.0.8
+  timezone: UTC
+  pvReclaimPolicy: Delete
+  pd:
+    baseImage: pingcap/pd
+    replicas: 3
```

Comment:
how about deploying 1 replica for pd/tikv/tidb to make this basic example can be applied in one node cluster, e.g. default cluster started by minikube / kind?

Reply:
I prefer 2/3/3, which preserves the taste of distributed and scalable